pkg/server/proto/server.proto

```diff
@@ -4749,6 +4759,28 @@ message Pipeline {
   }
 }
 
+message PipelineRun {
+  string id = 1;
+  // The sequence number for this pipeline run.
+  uint64 sequence = 2;
+
+  Ref.Pipeline pipeline = 3;
+
+  repeated Ref.Job jobs = 4;
+
+  State status = 5;
```

Comment:
I wonder if we should call this field `state`, mainly to be consistent with the Job message.

Reply:
Good call